src/components/Callout/Callout.tsx

```diff
@@ -17,8 +18,18 @@ export const Callout = ({
       variation="filled"
       colorTheme={warning ? 'warning' : 'info'}
       backgroundColor={backgroundColor}
+      hasIcon={false}
     >
-      <View>{children}</View>
+      <Flex>
+        <div className="amplify-message__icon">
+          {warning ? (
+            <IconWarning aria-hidden={false} aria-label="warning icon" />
```

Comment:
```suggestion
            <IconWarning aria-hidden={false} aria-label="Warning" />
```

These can just say "Warning/Important information". For alt text or aria-labels, you *usually* don't need to add "image of" "icon", etc. In this case we're conveying the context of what the Callout is; that it's important information, or a warning to the developer, not so much that an icon is present.